### PR TITLE
Mobile updates

### PR DESCRIPTION
**Pull request by dirkhh**
### Describe the pull request:

- [x] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

This combines two changes - too lazy to do two PRs

### Changes made:

1) fix several bugs in the handling of incorrect or unverified cloud credentials in the mobile app
2) attempt to better handle the UI layout on the rare phones where the default font is so big that we just can't get the UI to fit the screen without reducing font size

### Additional information:

This has been tested quite a bit with mobile on desktop, but has received very little testing on actual devices (since I don't have access to a device with such an odd font size / screen ratio)

### Release note:

added CHANGELOG

### Documentation change:

While the changes are user visible, they aren't something that needs to go into the user manual

### Mentions:

@bstoeger

### CODE REVIEW

**Comment by dirkhh** [comment]
Did anyone have a chance to look at this one?

File: mobile-widgets/qmlmanager.cpp [hunk 1]
```diff
@@ -457,9 +457,11 @@ void QMLManager::saveCloudCredentials()
 {
 	QSettings s;
 	bool cloudCredentialsChanged = false;
+	bool noCloud = QMLPrefs::instance()->credentialStatus() == qPrefCloudStorage::CS_NOCLOUD;
```
Comment (bstoeger) [inline mobile-widgets/qmlmanager.cpp:465]: Typo in commit message: `enry page`.

**Comment by bstoeger** [comment]
Sorry, apart from a trivial typo this is way outside my comfort zone. If it works for you, I'm fine with it.

**Comment by dirkhh** [comment]
We so need others to help with the QML client... ARGL.

Thanks for looking.

**Comment by dirkhh** [comment]
fixed locally and pushed to master

**Pull request closed without merging**